src/test/java/org/galatea/starter/entrypoint/BaseRestControllerTest.java

```diff
@@ -0,0 +1,42 @@
+package org.galatea.starter.entrypoint;
+
+import junitparams.JUnitParamsRunner;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+
+import static org.junit.Assert.assertEquals;
+
+@RequiredArgsConstructor
+@Slf4j
+// Use this runner since we want to parameterize certain tests.
+// See runner's javadoc for more usage.
+@RunWith(JUnitParamsRunner.class)
+public class BaseRestControllerTest extends BaseRestController {
```

Comment:
Formatting seems inconsistent in file. Plus make TARGET_ENCODED_MESSAGE private.